python/docs/dlis/metadata.rst

```diff
@@ -92,6 +92,39 @@ to make it easy to work with programmatically:
    >>> channel.long_name
    'Bond Index'
 
+Units
+-----
+
+Many dlis attributes have units stored alongside the value, but, unfortunately,
+dlisio lacks official support for them.
+But while they are not accessible through any public interface, they still are
+reachable.
+
+.. warning::
+   Please note that further information discloses dlisio internals, thus is
```

Comment:
Don't mention dlisio internals. The attic is public.
Would probably drop the entire warning tbh, we are still in pre v1.0.0 and everything can in theory be broken between versions (even `dlisio.load` :'D )